### PR TITLE
9 review language piscine rustroad c 396

### PR DESCRIPTION
**Pull request by davhojt**
Language review of road_intersection raid

### CODE REVIEW

File: subjects/road_intersection/README.md [hunk 1]
```diff
@@ -2,119 +2,107 @@
...
-  - `Up` south to north.
-  - `Down` north to south.
-  - `Right` west to east.
-  - `Left` east to west.
-
-- The `R` key must generate random vehicles with random lanes and routes.
+You will use your keyboard to spawn vehicles for your simulation. You will use the arrow keys to spawn a vehicle from an initial direction, on the appropriate side of the road, and with a random route:
 
-- The `Esc` key must finish the simulation.
+- **`↑` Up:** south to north.
```
[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/road_intersection/README.md:86]: or maybe just add 
**`↑` Up:** direction south to north.

**Review by MSilva95 — CHANGES_REQUESTED** [review]
I just feel like we could explain a little bit the arrows part, other than that its a go...

File: subjects/road_intersection/README.md [hunk 2]
```diff
@@ -2,119 +2,107 @@
...
+→ →                                     → →
+_______________             _______________
+               |     |     |
+               |     |     |
+               |     |     |
+               |     |     |
+               |     |     |
+               |  ↓  |  ↑  |
+               |  ↓  |  ↑  |
+```
```
[the remark this answers is not in the document]
Comment (davhojt) [reply subjects/road_intersection/README.md:47]: The problem is that it road really wide which makes it look like it has many lanes in each direction. I'll leave it like this, but perhaps it is better to replace it entirely with an image.

File: subjects/road_intersection/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (davhojt) [reply subjects/road_intersection/README.md]: Agreed. I made a change.

File: subjects/road_intersection/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/road_intersection/README.md:47]: its ok like this, if they read carefully they will get it

[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/road_intersection/README.md:47]: its ok like this, if they read carefully they will get it

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**